src/Engine/ProtoCore/DSASM/InstructionSet.cs

```diff
@@ -734,6 +734,7 @@ public bool TryGetArrayKey(out StackValue array, out int index)
         /// <returns></returns>
         public StackValue ToBoolean(Core core)
         {
+            RuntimeCore runtimeCore = core.__TempCoreHostForRefactoring;
```

Comment:
_tempcoreHostforRefactoring is temporary, the last PR will remove all _tempcoreHostforRefactoring.

The runtimeMemory was placed in RuntimeCore because only the VM should ever refer to it by design